pkg/remote/kyma_synchronization_context.go

```diff
@@ -167,7 +167,7 @@ func (c *KymaSynchronizationContext) CreateOrFetchRemoteKyma(
 	recorder := adapter.RecorderFromContext(ctx)
 	remoteKyma := &v1beta2.Kyma{}
 
-	remoteKyma.Name = kyma.Name
+	remoteKyma.Name = v1beta2.DefaultRemoteKymaName
```

Comment:
if fetch with defaultname -> found -> 
                                          -> not found ->  fallback to fetch with kyma.Name
                                                                                       found ->  update with defaultname
                                                                                       notfound ->  created with defaultname